pkg/webui/console/containers/fetch-select/index.js

```diff
@@ -33,11 +33,12 @@ export default function({
   defaultTitle,
   optionsFormatter = formatOptions,
   defaultDescription,
+  additionalOptions,
```

Comment:
```suggestion
  additionalOptions = [],
```

This breaks e2e tests as not every usage of `CreateFetchSelect` has this parameter defined.